inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/responses/ping/Trigger.kt

```diff
@@ -7,14 +7,14 @@ import com.google.gson.annotations.SerializedName
  */
 internal data class Trigger(
     @SerializedName("type")
-    val type: Int? = null,
+    val type: Int,
 
     @SerializedName("eventType")
     val eventType: Int? = null,
```

Comment:
The specs say that it's required only for `type` 1 (Event) but at the same time specs say that `1` is always returned. So I think we can set it to non-optional

Reply:
updated